nixos/modules/services/hardware/pcscd.nix

```diff
@@ -61,8 +61,8 @@ in {
       description = "PCSC-Lite daemon";
       environment.PCSCLITE_HP_DROPDIR = pluginEnv;
       serviceConfig = {
-        ExecStart = "${pkgs.pcsclite}/sbin/pcscd -f -x -c ${cfgFile}";
-        ExecReload = "${pkgs.pcsclite}/sbin/pcscd -H";
+        ExecStart = "${stdenv.lib.getBin pkgs.pcsclite}/sbin/pcscd -f -x -c ${cfgFile}";
+        ExecReload = "${stdenv.lib.getBin pkgs.pcsclite}/sbin/pcscd -H";
```

Comment:
Here and elsewhere `lib` is already available. Could you use `lib` instead of `stdenv.lib`?

Reply:
Will do. By the way, do you by any chance know why are packages encouraged to use `stdenv.lib` instead of `lib` but the converse is used for modules?